src/lexer/token.h

```diff
@@ -17,6 +17,73 @@ typedef enum {
     TT_CBRACKET, // ]
     TT_SEMI, // ;
     TT_NO_TOKEN, // Not a token
+	
+	TT_PERIOD, // .
+	TT_COMMA, // ,
+	TT_MINUS, // -
+	TT_PLUS, // +
+	TT_STAR, // *
+	TT_SLASH, // /
+	TT_ASSIGN, // =
+	TT_COLON, // :
+	TT_MOD, // %
+	TT_BAND, // &
+	TT_LAND, // &&
+	TT_BOR, // |
+	TT_LOR, // ||
+	TT_DEC, // -=
+	TT_INC, // +=
+	TT_PLUSPLUS, // ++
+	TT_MINUSMINUS, // --
+	TT_DIVEQ, // /=
+	TT_MULEQ, // *=
+	TT_MODEQ, // %=
+	TT_BANDEQ, // &=
+	TT_BOREQ, // |=
+	TT_LANDEQ, // &&=
+	TT_LOREQ, // ||=
+	TT_GREATER, // >
+	TT_LESS, // <
+	TT_LESSEQ, // <=
+	TT_GREATEREQ, // >=
+	TT_LEFTSHIFT, // <<
+	TT_RIGHTSHIFT, // >>
+	TT_LNOT, // !
```

Comment:
Also this is minor, but tilde is a bitwise NOT, so if we want to have naming consistent we could call them LNOT and BNOT respectively.